modules/installation-osp-creating-sr-iov-compute-machines.adoc

```diff
@@ -0,0 +1,195 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user-sr-iov.adoc
+// * installing/installing_openstack/installing-openstack-user-sr-iov-kuryr.adoc
+//
+// TODO: Get https://github.com/shiftstack/SRIOV-Compute-Nodes-Ansible-Automation into a supported
+//       repo, associate playbooks with individual releases, and then embed here.
+
+[id="installation-osp-creating-sr-iov-compute-machines_{context}"]
+= Creating compute machines that run on SR-IOV networks
+
+After standing up the control plane, create compute machines that run on the SR-IOV networks that you created in "Creating SR-IOV networks for compute machines."
```

Comment:
that s/."/". thing here and in the rest of the PR

Reply:
```suggestion
After standing up the control plane, create compute machines that run on the SR-IOV networks that you created in "Creating SR-IOV networks for compute machines".
```